tests/test_inventory_client.py

```diff
@@ -308,6 +415,43 @@ def test_add_asset(cli, init_assets):
     final_assets = init_assets + [created_asset]
     assert compare_unsorted_list(cli.assets(), final_assets, lambda x: x.vid)
 
+    asset_universe = cli.universe_of(created_asset.vid)
+    assert asset_universe.namespace == CURRENT_UNIVERSE.namespace
+    assert asset_universe.version == CURRENT_UNIVERSE.version.sem_version
+
+
+def test_add_asset_old_universe(cli, init_assets):
+    """Tests the method ``add_asset`` of the class ``InventoryClient``, when the
+    asset already exits but it's linked to an old version of the Universe"""
+    asset_id = init_assets[0].asset_id
+    asset = Asset(asset_id)
+    # Bump the current universe version and add an existing asset again.
+    next_version = UniverseVersion.from_int_version(
+        CURRENT_UNIVERSE.version.int_version + 1
+        )
+    old_universe_version = CURRENT_UNIVERSE.version
+    CURRENT_UNIVERSE.version = next_version
+    cli.g().ensure_universe().next()
+    timestamp = datetime.fromisoformat('2022-02-01T01:00:00+00:00')
+    expiration = datetime.fromisoformat('2022-02-07T01:00:00+00:00')
+
+    created_asset = cli.add_asset(asset, expiration, timestamp)
+
+    assert created_asset.vid is not None
+    assert created_asset.asset_id.type == asset.asset_id.type
+    assert created_asset.asset_id.identifier == asset.asset_id.identifier
+    assert created_asset.time_attr.first_seen == timestamp
+    assert created_asset.time_attr.last_seen == timestamp
+    assert created_asset.time_attr.expiration == expiration
+
+    asset_universe = cli.universe_of(created_asset.vid)
+    assert asset_universe.namespace == CURRENT_UNIVERSE.namespace
+    assert asset_universe.version == CURRENT_UNIVERSE.version.sem_version
```

Comment:
Shouldn't `sem_version` be a method call?

Reply:
No, not really look at the same answers above.